tests/test_scene.py

```diff
@@ -0,0 +1,622 @@
+from pythonbits import scene, bb
+from unittest.mock import patch, Mock, call
+import pytest
+
+import json
+import os
+import pickle
```

Comment:
Thank you for in fact writing tests, but I cannot accept pickle objects into the repository. This is inherently unsafe.

Reply:
What's wrong with pickle? How do you propose we store API responses for testing?

Reply:
A pickled object is just that, an object, which can contain almost arbitrary code, i.e. it is a (remote) code execution vulnerability as you can't even inspect it properly. The API responses could just be stored in plaintext HTML or JSON. If there is HTML then the common structure should of course be factored out.